python/tank/platform/engine.py

```diff
@@ -169,6 +171,13 @@ def __init__(self, tk, context, engine_instance_name, env):
         qt.QtGui = base_def.get("qt_gui")
         qt.TankDialogBase = base_def.get("dialog_base")
 
+        qt5_root = self.__define_qt5()
```

Comment:
does it make sense to make this part of `_define_qt_base`? Feels like a) we should follow the existing patterns and b) how does an engine user tweak qt5 properties to patch up any discrepancies?